opg-elasticsearch-shared-data-1604/docker-compose.yml

```diff
@@ -1,26 +0,0 @@
-elasticsearchshareddata:
```

Comment:
I don't think you want to remove the docker-compose files without replacing them with alternatives in the tests.
Without this config, you won't have a running example, so we shouldn't loose them yet